permission_handler_android/lib/src/permission_handler_android.dart

```diff
@@ -42,23 +17,27 @@ class PermissionHandlerAndroid extends PermissionHandlerPlatform {
     );
   }
 
+  /// Determine whether you have been granted a particular permission.
```

Comment:
Use a passive voice in the API documentation:
```suggestion
  /// Determine whether the application has been granted a particular permission.
```